src/id.js

```diff
@@ -14,6 +15,12 @@ module.exports = configure(({ ky }) => {
       searchParams: options.searchParams
     }).json()
 
-    return toCamel(res)
+    const output = toCamel(res)
+
+    if (output.addresses) {
+      output.addresses = output.addresses.map(ma => multiaddr(ma))
```

Comment:
Needs a BREAKING CHANGE commit message for this change as well if there isn't one already ;)

Reply:
...or just put this in the PR description and we can use it on squash&merge